geonode/upload/handlers/base.py

```diff
@@ -148,7 +148,16 @@ def extract_params_from_data(_data):
         Remove from the _data the params that needs to save into the executionRequest object
         all the other are returned
         """
-        return []
+        return {
+            "resource_pk": _data.pop("resource_pk", None),
+            "dataset_title": _data.pop("dataset_title", None),
+            "original_zip_name": _data.pop("original_zip_name", None),
+            "skip_existing_layers": _data.pop("skip_existing_layers", "False"),
+            "overwrite_existing_layer": _data.pop("overwrite_existing_layer", False),
+            "store_spatial_file": _data.pop("store_spatial_files", "True"),
+            "action": _data.pop("action", "upload"),
+            "custom": _data.pop("custom", {}),
+        }, _data
```

Comment:
Do we really need to move all these specific params names into the parent base class?

Reply:
Are these really that specific? I had the impression, that these are used by most of the handlers. Have a look, most of them were duplicated. In any case, if they are not used, they should not do any harm, do they?